acs/acs.go

```diff
@@ -0,0 +1,139 @@
+package acs
+
+import (
+	"sync/atomic"
+
+	"github.com/DE-labtory/cleisthenes"
+	"github.com/DE-labtory/cleisthenes/bba"
+	"github.com/DE-labtory/cleisthenes/pb"
+	"github.com/DE-labtory/cleisthenes/rbc"
+)
+
+type request struct {
```

Comment:
maybe it can separated with `acs.go` as `request.go` because `rbcMap`, `bbaMap` abstracted as `repository`

Reply:
no `request` in this case is inner message like BBA, RBC.